modules/custom/az_migration/migrations/migrate_plus.migration.az_event_categories.yml

```diff
@@ -41,3 +41,9 @@ process:
     source: '@parent_id'
 
   changed: timestamp
+
```

Comment:
Looks like we actually merged the event_categories migration as part of your last PR (?).  Did we actually want to preserve term IDs for event categories (line 16-17 of this file)?
```
process:
  tid: tid
```

Reply:
We dint. I have removed this line